src/client/components/common/Tabs.tsx

```diff
@@ -103,6 +103,7 @@ export interface TabPanelProps extends StyledSystemProps {
 }
 
 const StyledTabPanel = styled.div<{ hidden?: boolean }>`
+  ${({ hidden }) => (hidden ? 'display: none;' : 'display: flex;flex: 1;')}
```

Comment:
@FiboApe missing space after `;`. I like to not use the ternary tho and just do more lines in this case